extensions/sql-database-projects/src/dialogs/publishOptionsDialog.ts

```diff
@@ -157,8 +157,8 @@ export class PublishOptionsDialog {
 		this.optionsModel.InitializeUpdateOptionsMapTable();
 
 		await this.updateOptionsTable();
-		this.optionsFlexBuilder.removeItem(this.optionsTable);
-		this.optionsFlexBuilder.insertItem(this.optionsTable, 0, { CSSStyles: { 'overflow': 'scroll', 'height': '65vh' } });
+		this.optionsFlexBuilder?.removeItem(this.optionsTable!);
+		this.optionsFlexBuilder?.insertItem(this.optionsTable!, 0, { CSSStyles: { 'overflow': 'scroll', 'height': '65vh' } });
```

Comment:
why does the optionsTable need to be removed and re-inserted? Does it not get updated?

Reply:
Yes, it needs to be removed and inserted, there isn't supporting method for updating the existing one! :(